src/main/kotlin/de/craftan/engine/gameflow/states/AwaitStructurePlacement.kt

```diff
@@ -0,0 +1,24 @@
+package de.craftan.engine.gameflow.states
+
+import de.craftan.CraftanGameAction
+import de.craftan.engine.CraftanGame
+import de.craftan.engine.gameflow.RoundState
+import de.craftan.engine.gameflow.actions.PlaceStructureAction
+import de.craftan.engine.map.graph.StructureInfo
+
+class AwaitStructurePlacement(
+    game: CraftanGame,
+    allowedStructures: Set<StructureInfo>? = null, // null is to be interpreted as all structures are allowed
```

Comment:
Because this is in the pre game, we should allow only street and settlement. Therefore is this nullability not needed